src/main/java/seedu/address/logic/parser/ShowCommandParser.java

```diff
@@ -0,0 +1,48 @@
+package seedu.address.logic.parser;
+
+import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+
+import java.util.Arrays;
+
+import seedu.address.logic.commands.ShowCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+import seedu.address.model.person.NameContainsKeywordsPredicate;
+
+/**
+ * Parses input arguments and creates a new ShowCommand object
+ */
+public class ShowCommandParser implements Parser<ShowCommand> {
+
+    /**
+     * Returns a boolean expression if argument is valid
+     * @param args Arguments that are passed into parser
+     */
+    public static boolean isValidArgument(String args) {
+        String trimmedArgs = args.trim();
+        String[] validargs = new String[] {"Mon", "Tues", "Wed", "Thurs", "Fri", "Sat", "Sun"};
```

Comment:
I think its implemented as all 3 chars, so it should be tue and thu, other than that lgtm

Reply:
Got it!